### ISSUE TITLE
Include some cool JS-projects 

### ISSUE THREAD

**Issue by sunilsj99** [Cybros/JavaScriptProjects#1]
Go through README.md and include those projects. You can contribute your best javascript projects

**Comment by prtk1910** [comment]
Hey @sunilsj99 I want to help you out on this.

**Comment by salujaharkirat** [comment]
Hey where to create a pull request for the project code? Should I create a new repo and share that? @sunilsj99

**Comment by vsc-github** [comment]
#3 Check out this pull request. 
I was thinking of adding an 'Inspiration' section, where we can add several codepen projects just for inspiration. Should I go ahead and create that issue?

**Comment by theamanbhargava** [comment]
Yep, go ahead and create an issue, if possible, submit a PR as well with the inspiration section.

**Comment by theamanbhargava** [comment]
@jinxac Fork the repo and then send a PR.

**Comment by salujaharkirat** [comment]
@theamanbhargava cool thanks

**Comment by sunilsj99** [comment]
Those who want to contribute make a separate folder and put the project files in that folder. folder name should be the name of your project or something similar @vsc-github @zoroarctic

**Comment by KartikShankhavaram** [comment]
@jinxac Refer to [this](https://github.com/Cybros/JavaScriptProjects/blob/master/CONTRIBUTING.md) before contributing.

**Comment by viditvarshney** [comment]
@sunilsj99 Here is a Cool JS Project Named **Reaction-timer** **#23**

**Comment by viditvarshney** [comment]
**@KartikShankhavaram @sunilsj99 @vsc-github** Look this pull request 
This is a amazing JS Project "Reaction-Timer" **#23**  
**Please merge it from @viditvarshney**

**Comment by dharmadeveloper108** [comment]
I added a thing I made on codepen. I hope it's ok :)

**Comment by arghac14** [comment]
@sunilsj99 can I add some chrome extensions made with js?

**Comment by arghac14** [comment]
@sunilsj99 Also can I add a js quiz game?

**Comment by shagun25** [comment]
can i add the projects a fancy login/logout page?

**Comment by viditvarshney** [comment]
@all @shagun25 @arghac14  You can contribute to HacktoberFest2020 [here](https://github.com/viditvarshney/TOH_HacktoberFest2020)
